torchvision/models/vgg.py

```diff
@@ -145,6 +147,8 @@ class VGG11_BN_Weights(WeightsEnum):
                     "acc@5": 89.810,
                 }
             },
+            "_flops": 7.609090,
```

Comment:
I was expecting this to be higher for the BN version.

Reply:
Seems to be the same up to a decimal, and for all vgg models.

```
VGG11_Weights.IMAGENET1K_V1                           |    7609090048 | 506.840077
VGG11_BN_Weights.IMAGENET1K_V1                        |    7609090048 | 506.881400
VGG13_Weights.IMAGENET1K_V1                           |   11308466176 | 507.545068
VGG13_BN_Weights.IMAGENET1K_V1                        |   11308466176 | 507.589627
VGG16_Weights.IMAGENET1K_V1                           |   15470264320 | 527.795678
VGG16_Weights.IMAGENET1K_FEATURES                     |   15470264320 | 527.801824
VGG16_BN_Weights.IMAGENET1K_V1                        |   15470264320 | 527.866207
VGG19_Weights.IMAGENET1K_V1                           |   19632062464 | 548.051225
VGG19_BN_Weights.IMAGENET1K_V1                        |   19632062464 | 548.142819
```

Reply:
Yeap, it's because the flops script doesn't estimate all ops. It covers mostly convs, linear layers etc which account for 99% of computations usually. Ops like aten.mean/aten.mul used in BN layers aren't factored in. Maybe on the future we can have a better more precise utility to estimate the flops but for now this will do. We reduced the precision for 3 decimal points as larger precision for the total giga flops/ips doesn't make too much sense.